package.json

```diff
@@ -26,7 +26,7 @@
     "gatsby": "4.20.0",
     "gatsby-plugin-catch-links": "4.20.0",
     "gatsby-plugin-google-gtag": "4.23.0",
-    "gatsby-plugin-manifest": "4.20.0",
+    "gatsby-plugin-manifest": "^4.20.0",
```

Comment:
What is this change about? I think renovate will complain about the caret if this gets merged.

https://docs.renovatebot.com/dependency-pinning/

Reply:
thanks, good point. I trusted the npm installer from gatsby.
didn't think about renovate.
will remove

Reply:
done